gh-actions/prepare-host/action.yaml

```diff
@@ -0,0 +1,25 @@
+name: 'Prepare host'
+description: 'Prepares the GitHub action runner for the submariner e2e testing'
+inputs:
+  cabledriver:
+    description: 'Install any system modules for specific cable drivers'
+    required: false
+    default: ''
+runs:
+  using: "composite"
+  steps:
+    - name: Reclaim free space
+      shell: bash
+      run: |
+        # cleanup 30GB+ of tools we don't need for our CI
+        rm -rf /opt/ghc /usr/share/dotnet /usr/share/swift
+        df -h
+        free -h
+
+    - name: Install WireGuard kernel module (only in WireGuard jobs)
+      shell: bash
+      run: |
+        if [[ "${{ inputs.cabledriver }}" == "wireguard" ]]; then
+          sudo apt install -y linux-headers-$(uname -r) wireguard
+          sudo modprobe wireguard
+        fi
```

Comment:
```suggestion
        [[ "${{ inputs.cabledriver }}" == "wireguard" ]] || exit
        sudo apt install -y linux-headers-$(uname -r) wireguard
        sudo modprobe wireguard
```

Reply:
I'm trying nested groups to see if it works... to make more informative output. If it doesn't work I'll use this suggestion.

Reply:
https://github.com/submariner-io/shipyard/pull/422/checks?check_run_id=1769053211#step:3:17 it doesn't do nesting properly, so... I'll apply

Reply:
Done.